docs/install.md

```diff
@@ -8,6 +8,31 @@
 
 * Important: If you intend to use the Volume Snapshot feature, the [Kubernetes Volume Snapshot CRDs](https://github.com/kubernetes-csi/external-snapshotter/tree/master/client/config/crd) must be installed **before** the EBS CSI driver. For installation instructions, see [CSI Snapshotter Usage](https://github.com/kubernetes-csi/external-snapshotter#usage).
 
+### Metadata
+
+The EBS CSI Driver uses a metadata source in order to gather necessary information about the environment to function. The driver currently supports two metadata sources: [IMDS](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) or Kubernetes.
+
+The controller `Deployment` can skip metadata if the region is provided via the `AWS_REGION` environment variable (Helm parameter `controller.region`). The node `DaemonSet` requires metadata and will not function without access to one of the sources.
+
+#### IMDS (EC2) Metadata
```

Comment:
This is great content, but I'm a bit worried that `install.md` is getting a bit long. Could we move these two sections `IMDS (EC2) Metadata` and `Kubernetes Metadata) to an FAQ section that is linked here? 

Especially because it seems like the metadata experience works by default for most customers.

Reply:
I'd like to push back on this - this is not a "Frequently Asked Question", it is part of the install